### DISCUSSION TITLE
Sortet Tablist by Permissions

### DISCUSSION THREAD

**Discussion by Skyfighter1** [Zrips/CMI#1480] [Feature Requests]
**Description of feature request:**

Hey could you add that the tablist is sortet? So I mean that the owner stand above player.... maybe you could do that with permissions

**Comment by TheWranger** [comment]
Already posted here: https://github.com/Zrips/CMI/issues/931 (would still like to see this added, though)

**Comment by Vehzx** [comment]
+! this, would be great to see
